utils/tb_log_parser.py

```diff
@@ -104,78 +118,102 @@ def parse_valid(self, line):
 
         # do not parse lines like
         # [2021-04-16 22:29:19] [valid] [valid] First sentence\'s tokens as scored:
-        if line.find('Ep.') == -1 and line.find('Up.') == -1:
+        if line.find("Ep.") == -1 and line.find("Up.") == -1:
             return
 
-        _, _, _, _ep, ep, _, _up, up, _, metric, _, value, _, _stalled, x, *_times = line.split()
```

Comment:
I added the line length to the ruff configuration, but not to the black configuration. I added it and re-ran the formatting. This fixes it so that it looks like the original.
I missed in the black configuration to also add the line limit. I set it to `99` similar to mozilla-central.

We can always opt out for places where it does the wrong thing with:

```
# fmt: off

...

# fmt: on
```

Reply:
> why is it adding braces and new lines

Also to answer the question directly, it was because it broke the line length limit, and black decided to re-format it.